crd/nodenetworkconfig/api/v1alpha/nodenetworkconfig.go

```diff
@@ -68,6 +68,14 @@ type NodeNetworkConfigStatus struct {
 	NetworkContainers []NetworkContainer `json:"networkContainers,omitempty"`
 }
 
+// NetworkContainerStatus defines the status of the Network Container
+type NetworkContainerStatus struct {
+	Timestamp metav1.Time `json:"timestamp,omitempty"`
```

Comment:
what's the purpose of including a timestamp?

Reply:
Updating the NC status based on the timestamp on the CNS side. You always want the latest status only to be reflected in CNS.

Reply:
Not sure I follow. The NNC should always contain the latest NC state, including the latest status. CNS should always act on whatever latest NNC state there is. How will this timestamp be used?

Reply:
I think I agree here. Removing Timestamp